packages/web/src/components/spa/HackAUBG/HackAUBG.jsx

```diff
@@ -31,42 +30,61 @@ export const HackAUBG = () => {
         new Anchor('FAQ', '#faq', <FaRegLightbulb />),
     ];
 
+    const match = useMatch('/hackaubg/verify');
+
+    const [showVerification, setShowVerification] = useState(!!match);
```

Comment:
doesn't useMatch return a boolean value? Why do you have to double negate?

Reply:
There is no reason yeah. Will be removed